core/tests/protractor/learnerDashboard.js

```diff
@@ -129,17 +133,17 @@ describe('Learner dashboard functionality', function() {
     // 'In Progress' section.
     player.submitAnswer('Continue', null);
     browser.ignoreSynchronization = true;
-    browser.get(general.LEARNER_DASHBOARD_URL);
+    learnerDashboardPage.get();
     general.acceptAlert();
     browser.waitForAngular();
     general.waitForSystem();
     browser.ignoreSynchronization = false;
-    element(by.css('.protractor-test-incomplete-collection-section')).click();
+    learnerDashboardPage.navigateToIncompleteCollection();
     browser.waitForAngular();
     general.waitForSystem();
-    expect(element.all(by.css(
-      '.protractor-test-collection-summary-tile-title')).first(
-    ).getText()).toMatch('Test Collection');
+    expect(
+      learnerDashboardPage.openSummaryTile()
```

Comment:
open...() doesn't have a connotation of "return a value". Maybe it should be "getTitleOfSummaryTile()" or similar, though that's just an example -- you'd need to check what exactly is being retrieved here and include that in the method name.

Reply:
Ok

Reply:
Done